### ISSUE TITLE
Broken crates.io badge in Readme

### ISSUE THREAD

**Issue by atouchet** [frewsxcv/rust-crates-index#67]
https://meritbadge.herokuapp.com/crates-index no longer loads. The solution for the crates.io badge would be to switch to https://img.shields.io/crates/v/crates-index.svg however the link itself was changed to direct to lib.rs and I don't know if there's a badge for that instead of crates.io.

**Comment by inomag** [comment]
![Capture](https://user-images.githubusercontent.com/66073063/136684132-5096f5ed-6f0e-4228-8f61-705030505800.JPG)

@atouchet, is this ok?

**Comment by atouchet** [comment]
I mean that would fix the crates.io badge but as it no longer actually links to crates.io I'm not sure if that's the desired result.

**Comment by inomag** [comment]
yes you are right. it links to https://crates.rs/crates/crates-index

**Comment by inomag** [comment]
![Screenshot (226)](https://user-images.githubusercontent.com/66073063/136684577-fb359b93-9eb3-4f7b-8c0f-c83f8c832ad9.png)

after changing the url it now links to crates.io
